--- scriptsdownload.pl	2019-12-01 054325.000000000 +0000
+++ scriptsdownload.pl	2019-12-01 060802.494788876 +0000
@@ -82,7 +82,7 @@
 	}
 
 	return $have_curl
-		 (qw(curl -f --connect-timeout 20 --retry 5 --location --insecure), shellwords($ENV{CURL_OPTIONS}  ''), $url)
+		 (qw(curl -f --connect-timeout 20 --max-time 240 --retry 5 --location --insecure), shellwords($ENV{CURL_OPTIONS}  ''), $url)
 		 (qw(wget --tries=5 --timeout=20 --no-check-certificate --output-document=-), shellwords($ENV{WGET_OPTIONS}  ''), $url)
 	;
 }